### PR TITLE
Add rule for using the Player CLI to compile XLRs

### PR DESCRIPTION
**Pull request by KetanReddy**
- Adds rule `xlr_compile` to use the Player CLI to compile XLRs for a package. 
- Didn't integrate with the `js_library_pipeline` since its not applicable in all use cases.

### CODE REVIEW

**Comment by KetanReddy** [comment]
Build failure is just due to a missing tag.

File: player/cli/xlr.bzl [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  output_depset = depset(file_outputs)
+
+  return [
+      DefaultInfo(
+          files = output_depset,
+      ),
+      JSModuleInfo(
+          direct_sources = output_depset,
+          sources = output_depset,
+      ),
```
Comment (adierkens) [inline player/cli/xlr.bzl:72]: Does this need a `JSModuleInfo` if this isn't an actual JS package?

Comment (KetanReddy) [reply player/cli/xlr.bzl:72]: I wasn't 100% sure since it does produce a js file that could be imported in other down-stream packages.

**Review by adierkens — APPROVED** [review]

**Comment by intuit-svc** [comment]
:rocket: PR was released in [`v0.10.0`](https://github.com/player-ui/rules_player/releases/tag/v0.10.0) :rocket:

**Pull request merged**